submissions/onyemeMichael/to-do-app/src/components/Todo.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import TodoForm from './TodoForm';
+import { RiCloseCircleLine } from 'react-icons/ri';
+import { TiEdit } from 'react-icons/ti';
+
+
+
+const Todo = ({ todos, completeTodo, removeTodo, updateTodo}) => {
+  const [edit, setEdit] = useState({
+    id: null,
+    value: ''
+  });
+
+  const submitUpdate = value => {
+    updateTodo(edit.id, value);
+    setEdit({
+      id: null,
+      value: ''
+    });
+  };
+
+  if (edit.id) {
+    return <TodoForm edit={edit} onSubmit={submitUpdate} />;
+  };
+
+ console.log(todos);
+  return todos.map((todo, index) => (
+    <div
+      className={todo.isComplete ? 'todo-row complete' : 'todo-row'}
+      key={index}
+    >
+      <div key={todo.id} onClick={() => completeTodo(todo.id)} style={{ textDecoration: todo.isCompleted ? "line-through" : "",color: todo.isCompleted ? "" : ""}}>
+        {todo.text}
+      </div>
+      <div className='icons'>
+        <RiCloseCircleLine
+          onClick={() => removeTodo(todo.id)}
+          className='delete-icon'
+        />
+        <TiEdit
+          onClick={() => setEdit({ id: todo.id, value: todo.text })}
+          className='edit-icon'
+        />
+        <div>
+          <label className='container'>
+          <input type="checkbox"  onClick={() => completeTodo(todo.id)}/>
+          <span className="checkmark"></span>
```

Comment:
This should be self-closing

Reply:
Pls explain further.

Reply:
If  your html tag does not have content in react, you make it self closing. In this case, `<span className="checkmark" />` this is what you should have. @MYKEL7